tomcat7/image.yaml

```diff
@@ -159,9 +159,7 @@ modules:
 packages:
       repositories:
           - jboss-os
-          - jboss-rhscl
```

Comment:
You cannot remove this, `rh-mongodb32-mongo-java-driver` is installed from this repository.